libpretixsync/build.gradle

```diff
@@ -118,3 +121,19 @@ task copyTestResources(type: Copy) {
     into "${buildDir}/classes/test"
 }
 processTestResources.dependsOn copyTestResources
+
+sqldelight {
+    databases {
+        SyncDatabase {
+            packageName = "eu.pretix.libpretixsync.sqldelight"
+            srcDirs('src/main/sqldelight/sqlite', 'src/main/sqldelight/common', 'src/main/sqldelight/migrations')
+
+            version = 105
```

Comment:
I'm wondering if this is needed at all, since the version it deducts from the migration files seems to take precedent? I'll check if we can just remove it. If not, I agree that it should go into a common file